src/paid-cookie-consent/guidance-handler.js

```diff
@@ -56,7 +56,12 @@ export default async function handler(message, context) {
     url,
     guidanceParsed,
   );
-  await Suggestion.create(suggestionData);
+  const requiresValidation = Boolean(context.site?.requiresValidation);
+
+  await Suggestion.create({
+    ...suggestionData,
+    status: requiresValidation ? Suggestion.STATUSES.NOT_VALIDATED : Suggestion.STATUSES.NEW,
```

Comment:
why do you need to add status here when you already added it at:
https://github.com/adobe/spacecat-audit-worker/blob/105b791f483873520c264a040a8208aba05c6e41/src/paid-cookie-consent/guidance-opportunity-mapper.js#L140